source/tyr/tyr/tasks.py

```diff
@@ -235,6 +235,25 @@ def import_autocomplete(files, autocomplete_instance, async=True, backup_file=Tr
 
 
 @celery.task()
+def import_in_mimir(files, instance, async=True):
+    """
+    Import pt data stops to autocomplete
+    """
+    current_app.logger.debug("Import pt data to mimir")
+    actions = []
+    instance_config = load_instance_config(instance.name)
+
+    for _file in files:
```

Comment:
I don't really think that this makes sens to pass a function here, I think it's better to check beforehand that there is only one element and give only one file here
and thus you can just say (check if this works)

```python
    action = stops2mimir.si(instance_config, _file, None)
    if async:
        return actions.delay()
    else:
        # all job are run in sequence and import_in_mimir will only return when all the jobs are finish
        return action.apply()
```